Frontend/src/Components/CartScreen/CartScreenSlice.ts

```diff
@@ -0,0 +1,53 @@
+import { createSlice } from "@reduxjs/toolkit";
+interface productProps {
```

Comment:
Types should start with a capital letter. 
```suggestion
interface ProductProps {
```